src/test/java/io/confluent/examples/streams/window/DailyTimeWindows.java

```diff
@@ -52,16 +52,6 @@ public DailyTimeWindows(final ZoneId zoneId, final int startHour, final Duration
         return windows;
     }
 
-    @Override
-    public long maintainMs() {
```

Comment:
@jeanlouisboudart @csarrazi 

This method should not be overwritten. Instead, the store retention time should be increase by the user of `DailyTimeWindows`